sites/skeleton.dev/src/examples/guides/cookbook/chat/Example.svelte

```diff
@@ -135,7 +134,7 @@
 								: 'bg-surface-hover-token'}"
 							onclick={() => (currentPersonId = person.id)}
 						>
-							<Avatar src="https://i.pravatar.cc/?img={person.avatar}" name={person.name} size="size-8" />
+							<!-- <Avatar src="https://i.pravatar.cc/?img={person.avatar}" name={person.name} size="size-8" /> -->
```

Comment:
We might be shooting ourselves in the foot to purge all v3 components before replacing with the v4 version, as we may have to disable a wide number of components in the docs. It might be better to purge these one-by-one and do a sweep through the docs to update to the new standards.

Otherwise our docs may not be usable for a while. Depends on how many components we use. I know there's a ton of on the homepage.

Let's review on Discord.